packages/babel-generator/src/buffer.js

```diff
@@ -47,17 +47,17 @@ export default class Buffer {
 
   append(str: string): void {
     this._flush();
-    this._append(str, this._sourcePosition.line, this._sourcePosition.column,
-      this._sourcePosition.filename);
+    const { line, column, filename } = this._sourcePosition;
```

Comment:
Good call, I was probably overly worried about perf when I skipped the temp variables.